### DISCUSSION TITLE
GUI database style view for metadata

### DISCUSSION THREAD

**Discussion by krp312** [beetbox/beets#5092] [Q&A] [answered]
Greetings,

I'm currently dealing with the bug detailed in [this PR](https://github.com/beetbox/beets/pull/4582/).

@JOJ0 referring to the screenshot below, what tool are you using? I might have similarly malformed data for  _releasetype_ as well. I'd like to view my library's metadata in this way so I can quickly scroll to see what tracks are affected, and see what else might be wrong with other fields. Not sure how to do this gracefully with beets. (If there is a better way to do this, I'm all ears.)

> ![image](https://user-images.githubusercontent.com/2733783/220386096-d0894d62-7511-4cb0-aaea-6a9db9b627f8.png)

Many thanks in advance.

**Comment by JOJ0** [comment, marked as answer]
First of all: What beets version? Please use latest git master.

Tool is "DB Browser for SQLite"

**Reply by krp312** [reply]
thanks, def. using 1.6.0 with the latest master ([this](https://hub.docker.com/r/linuxserver/beets))

**Reply by krp312** [reply]
Thank you again! This is so good!!
